packages/world/src/systems/MineSystem.sol

```diff
@@ -198,7 +198,29 @@ contract MineSystem is System {
     }
   }
 
-  function _destroyEntity(EntityId caller, EntityId mined, ObjectType minedType, Vec3 baseCoord) internal {
+  function _handleAbove(EntityId caller, Vec3 coord) internal {
+    // Remove growables on top of this block
+    Vec3 aboveCoord = coord + vec3(0, 1, 0);
+
+    (EntityId above, ObjectType aboveType) = EntityUtils.getOrCreateBlockAt(aboveCoord);
+
```

Comment:
I feel like this should remain a `getBlockAt` instead of `getOrCreateBlockAt`

If you recall, if we keep it `getOrCreateBlockAt`, then on every mine you will be creating an extra entity, so 3 extra table writes